src/Components/ProductListingPage/FilterSection/RatingSection/RatingSection.jsx

```diff
@@ -1,54 +1,49 @@
 import React from "react";
 import { useFilter } from "../../../../Context";
-import InputTypeTwo from "../../../UI/Input/InputTypeTwo";
+import "./RatingSection.css";
 
 const RatingSection = () => {
-  const { filterState, filterDispatch } = useFilter();
-  const { oneStar, twoStar, threeStar, fourStar } = filterState.byRating;
+  const {
+    filterState: { byRating },
+    filterDispatch,
+  } = useFilter();
+
+  const onRatingChangeHandler = (e) => {
+    filterDispatch({ type: "ratings", payload: e.target.value });
+  };
 
   return (
     <div className="rating">
       <h2>Ratings</h2>
-      <InputTypeTwo
-        inputWrapper="radio-input"
-        type="radio"
-        name="radio"
-        checked={fourStar}
-        label="4 star & above"
-        onChange={() => {
-          filterDispatch({ type: "4+", payload: "4+" });
-        }}
-      />
-      <InputTypeTwo
-        inputWrapper="radio-input"
-        type="radio"
-        name="radio"
-        checked={threeStar}
-        label="3 star & above"
-        onChange={() => {
-          filterDispatch({ type: "3+", payload: "3+" });
-        }}
-      />
-      <InputTypeTwo
-        inputWrapper="radio-input"
-        type="radio"
-        name="radio"
-        checked={twoStar}
-        label="2 star & above"
-        onChange={() => {
-          filterDispatch({ type: "2+", payload: "2+" });
-        }}
-      />
-      <InputTypeTwo
-        inputWrapper="radio-input"
-        type="radio"
-        name="radio"
-        checked={oneStar}
-        label="1 star & above"
-        onChange={() => {
-          filterDispatch({ type: "1+", payload: "1+" });
-        }}
-      />
+      <div className="rating-slider-label">
+        <span className="rating-badge-star">
+          <i className={byRating.oneStar ? "fas fa-star" : "far fa-star"}></i>
+        </span>
+        <span className="rating-badge-star">
+          <i className={byRating.twoStar ? "fas fa-star" : "far fa-star"}></i>
+        </span>
+        <span className="rating-badge-star">
+          <i className={byRating.threeStar ? "fas fa-star" : "far fa-star"}></i>
+        </span>
+        <span className="rating-badge-star">
+          <i className={byRating.fourStar ? "fas fa-star" : "far fa-star"}></i>
+        </span>
+        <span className="rating-badge-star">
+          <i className={byRating.fiveStar ? "fas fa-star" : "far fa-star"}></i>
+        </span>
+      </div>
```

Comment:
Appreciation: Now the Rating component has become much more pleasing to both the user and developer (less code and easy to manage).

Suggestion: You could make a small component for each star here as most of the values looks repetitive. Reducing 10 lines of code to 2-3 lines